src/TuDa.CIMS.Shared/Dtos/CreateSolventDto.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace TuDa.CIMS.Shared.Dtos;
+
+public class CreateSolventDto : CreateChemicalDto { }
```

Comment:
⛏️ Use empty class declaration
```suggestion
public class CreateSolventDto : CreateChemicalDto;
```